static/css/components/book.less

```diff
@@ -22,6 +22,9 @@
     a {
       margin: auto;
```

Comment:
Let's move the `width: 100%` declaration here, and delete the `a` selector that you added.